.codecov.yml

```diff
@@ -12,3 +12,8 @@ coverage:
                                #   option: "X%" a static target percentage to hit
         if_not_found: success  # if parent is not found report status as success, error, or failure
         if_ci_failed: error    # if ci fails report status as success, error, or failure
+
+    patch:
+      default:
+        enabled: yes
+        target: 95
```

Comment:
you might want to drop the total 98 target as well

Reply:
The full project is still over 98 so we can keep it for now IMO. If we find that >2% of the project actually contains unused code, we can drop the coverage target there too.